service/templates/logout.html

```diff
@@ -1,23 +1,41 @@
-{% extends 'base.html' %}
-{% block content %}
- <div class="container">
-    <h2> Tapis Logout </h2>
- </div>
-
-{% if logout_message %}
-<p>{{ logout_message}}</p>
-
-{% else %}
- <form action="" method="post">
+{% extends 'base-auth.html' %}
 
+{% block content %}
+<main class="s-form-page">
+  <form action="" method="post" class="s-form s-form--login">
+    <h1>
+      <figure>
+      {% with tenant_id=tenant_id %}
+      {% include "./branding.html" %}
+      {% endwith %}
+      {% with tenant_id="tacc" %}
+      {% include "./branding.html" %}
+      {% endwith %}
+      </figure>
+    </h1>
 
- <div class="container">
-        <label>
-          <input type="checkbox" checked="checked" name="logout"> Logout
-        </label>
-        <button type="submit">Submit</button>
-     </div>
-
-</form>
-{% endif %}
-{% endblock %}
\ No newline at end of file
+    {% if logout_message %}
+    <p>
+      {{ logout_message }}
+    </p>
+    <style>
+    .s-form p:only-child {
+      text-align: center;
+      margin-block: unset;
+    }
+    </style>
+    {% else %}
+    <p>Are you sure you want to log out?</p>
+    <input type="hidden" checked="checked" name="logout">
```

Comment:
**Oddness.** The `checked="checked" is wrong and useless. In fact, I don't think this is necessary.

```suggestion
```

And, if I delete this, then I can delete the comment ([from `controllers.py` line 1886](https://github.com/tapis-project/authenticator/blob/dev/service/controllers.py#L1886))`# if they submitted the logout form but did not check the box then just return them to the logout form -`.

But what server-side code should I also delete.... was that comment even accurate? I do not see code that checks the value of the `name="logout"` form field.

Reply:
Yup. The `request.form.get("logout")` is where the checkbox value is compared. Work to do before merge.

Reply:
Fixed (I think) via 69e4031.

Reply:
Hm, I use a different fix, 79b0f84, that does not require editing server-side code. _Now button and checkbox are truly merged. User must have clicked that button (or passed that POST data) to logout._